templates/CRM/Case/Form/ActivityTab.tpl

```diff
@@ -10,10 +10,10 @@
 {*this template is used for activity accordion*}
 {assign var=caseid value=$caseID}
 {if $isForm}
-  <div class="crm-accordion-wrapper crm-case_activities-accordion  crm-case-activities-block">
-    <div class="crm-accordion-header">
+  <details class="crm-accordion-bold crm-case_activities-accordion  crm-case-activities-block" open>
+    <summary>
       {ts}Activities{/ts}
-    </div>
+    </summary>
 
     <div id="activities" class="crm-accordion-body padded">
     <div class="crm-collapsible crm-search_filters-accordion collapsed">
```

Comment:
There's another accordion here (pattern 3), light bg. Worth swapping for `<details class="crm-accordion-light">… etc`